File: CotuTalk_Program/app/src/main/java/com/example/cotutalk_program/AcessoAPI/data/Usuario.kt
package com.example.cotutalk_program.AcessoAPI.data

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Usuario(
    val idUsuario : Int,
    val nome : String,
    val email : String,
    val senha : String,
    val biografia : String,
    val imagePath : String
): Parcelable

File: api/TWTodos/Models/Usuario.cs
using System.Text.Json.Serialization;

public class Usuario
{
    [JsonPropertyName("idUsuario")]
    public int IdUsuario { get; set; }

    [JsonPropertyName("nome")]
    public string Nome { get; set; }

    [JsonPropertyName("email")]
    public string Email { get; set; }

    [JsonPropertyName("senha")]
    public string Senha { get; set; }

    [JsonPropertyName("biografia")]
    public string Biografia { get; set; }

    [JsonPropertyName("imagePath")]
    public string ImagePath { get; set; }
    [JsonIgnore]
    public ICollection<Membro> Membros { get; set; } = new List<Membro>();
    [JsonIgnore]
    public ICollection<Postagem> Postagens { get; set; } = new List<Postagem>();
    [JsonIgnore]
    public ICollection<Curtida> Curtidas { get; set; } = new List<Curtida>();
    [JsonIgnore]
    public ICollection<Resposta> Respostas { get; set; } = new List<Resposta>();
}
